src/nitclk/metadata/config.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# This file is generated from NI-TClk API metadata version 255.0.0d0
+config = {
+    'api_version': '255.0.0d0',
+    'c_function_prefix': 'niTClk_',
+    'close_function': None,
+    'context_manager_name': {
+        'abort_function': None,
+        'initiate_function': 'Initiate',
+        'task': 'acquisition'
+    },
+    'custom_types': [
+    ],
+    'driver_name': 'NI-TClk',
+    'init_function': 'InitForDocumentation',
+    'library_info': {
+        'Linux': {
+            '64bit': {
+                'name': 'libnitclk.so',
+                'type': 'cdll'
+            }
+        },
+        'Windows': {
+            '32bit': {
+                'name': 'nitclk.dll',
+                'type': 'windll'
+            },
+            '64bit': {
+                'name': 'nitclk_64.dll',
+                'type': 'cdll'
+            }
+        }
+    },
+    'metadata_version': '2.0',
+    'module_name': 'nitclk',
+    'repeated_capabilities': [
+    ],
+    'session_class_description': 'An NI-TClk session.',
```

Comment:
Since there isn't an NI-TClk sessions, I'd leave this blank or even remove it if possible.

Reply:
This is not really true though. The TClk functions take a list of TClk sessions. True, they don't come from instantiating a tclk session object, but they do exist.

Reply:
> The TClk functions take a list of TClk sessions.

I thought they take a list of `nitclk.SessionReference` objects.
Is that what `'session_class_description'` is going to document? If so, it needs to be adjusted. It's a reference to an instrument / driver session, not an NI-TClk session.

Reply:
You're right. I was confusing where this was used.

Removed.